### ISSUE TITLE
change keras required version to 0.3.2

### ISSUE THREAD

**Issue by MInner** [farizrahman4u/seq2seq#49]
hi,

for some reason when I try to install it via pip it downgrade keras to 0.3.1 and it does not seem to work well with it, because keras.backend.rnn has argument "masking" in pip's keras 0.3.1 and you are using "maks" as in new 0.3.2

Ben

**Comment by farizrahman4u** [comment]
Done
